modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/order/engine/test/CommerceOrderEngineTest.java

```diff
@@ -1005,19 +999,12 @@ public void testGetNextOrderStatusesWhileOrderNotOpen() throws Exception {
 			PendingCommerceOrderStatusImpl.KEY,
 			_commerceOrder.getOrderStatus());
 
-		List<CommerceOrderStatus> nextCommerceOrderStatuses =
-			_commerceOrderEngine.getNextCommerceOrderStatuses(_commerceOrder);
-
-		Stream<CommerceOrderStatus> stream = nextCommerceOrderStatuses.stream();
-
-		List<CommerceOrderStatus> inProgressCommerceOrderStatuses =
-			stream.filter(
-				entry -> entry.getPriority() == -1
-			).collect(
-				Collectors.toList()
-			);
-
-		Assert.assertFalse(inProgressCommerceOrderStatuses.isEmpty());
+		Assert.assertTrue(
+			ListUtil.exists(
+				_commerceOrderEngine.getNextCommerceOrderStatuses(
+					_commerceOrder),
+				commerceOrderStatus ->
+					commerceOrderStatus.getPriority() == -1));
```

Comment:
Changed here:
assert that at least 1 CommerceOrderStatus satisfy entry.getPriority() == -1